.github/workflows/csharp-build.yml

```diff
@@ -46,6 +46,8 @@ on:
 jobs:
   build:
```

Comment:
can you make this one build-and-test for consistency plz

Reply:
:+1: